cpp/oneapi/dal/algo/dbscan/backend/gpu/kernel_fp_impl.hpp

```diff
@@ -138,14 +153,19 @@ struct get_core_narrow_kernel {
                                     data_ptr[j * column_count + i];
                         sum += val * val;
                     }
-                    if constexpr (use_weights) {
-                        count += sum <= epsilon ? weights_ptr[j] : count_type(0);
+                    if (sum > epsilon) {
```

Comment:
would if(epsilon > sum) { break;} have the same effect? might be cleaner to have a breaking condition rather than a continue condition.

Reply:
Its a good point overall, but for narrow_kernel(when column count <= 4), we are using default for loop, and this continue means transition to next iteration, not breaking the loop.